app/src/js/__global.js

```diff
@@ -14,3 +14,27 @@ function showInDevelopmentOption () {
     $("#conditional-environment").hide();
   }
 }
+
+document.getElementById('restricted-ips').style.display='none';
+document.getElementById('js-domains').style.display='none';
+document.getElementById('redirect-uris').style.display='none';
+
+function restClick(){
+  document.getElementById('restricted-ips').style.visibility='visible';
+  document.getElementById('restricted-ips').style.display = 'block';
+  document.getElementById('js-domains').style.visibility='visible';
+  document.getElementById('js-domains').style.display = 'block';
+  document.getElementById('redirect-uris').style.display='none';
+}
+function streamClick(){
+  document.getElementById('restricted-ips').style.visibility='visible';
+  document.getElementById('restricted-ips').style.display = 'block';
+  document.getElementById('js-domains').style.display='none';
+  document.getElementById('redirect-uris').style.display='none';
+}
+function webClick(){
+  document.getElementById('restricted-ips').style.display='none';
+  document.getElementById('js-domains').style.display='none';
+  document.getElementById('redirect-uris').style.visibility='visible';
+  document.getElementById('redirect-uris').style.display = 'block';
```

Comment:
This looks a bit complicated. I would have a function that hides the divs
```
function clearDivs(){
  document.getElementById('restricted-ips').style.visibility='none';
  document.getElementById('redirect-uris').style.display='none';
  document.getElementById('js-domains').style.visibility='none';
}
```

Which you call in each function e.g.
```
function restClick(){
  clearDivs();
  document.getElementById('restricted-ips').style.visibility='visible';
  document.getElementById('js-domains').style.visibility='visible';
}
```
You can also call the function when the page loads to clear everything.

Reply:
Changed